src/ui/server/controllers/insurance/business/turnover/save-and-back/index.test.ts

```diff
@@ -0,0 +1,106 @@
+import { Request, Response } from '../../../../../../types';
+import { post } from '.';
+import { FIELD_IDS, ROUTES } from '../../../../../constants';
+import { mockReq, mockRes, mockApplication, mockExporterBusinessTurnover } from '../../../../../test-mocks';
+import mapAndSave from '../../map-and-save';
+
+const {
+  EXPORTER_BUSINESS: {
+    TURNOVER: { ESTIMATED_ANNUAL_TURNOVER, PERCENTAGE_TURNOVER },
+  },
+} = FIELD_IDS.INSURANCE;
+
+const { INSURANCE_ROOT, ALL_SECTIONS } = ROUTES.INSURANCE;
+describe('controllers/insurance/business/turnover/save-and-back', () => {
+  let req: Request;
+  let res: Response;
+
+  let updateMapAndSave = jest.fn(() => Promise.resolve(true));
+
+  beforeEach(() => {
+    req = mockReq();
+    res = mockRes();
+
+    res.locals.application = mockApplication;
+
+    mapAndSave.turnover = updateMapAndSave;
+  });
+
+  afterAll(() => {
+    jest.resetAllMocks();
+  });
+
+  describe('post - save and back', () => {
+    describe('when there are no validation errors', () => {
+      it('should redirect to all sections page', async () => {
+        req.body = {
+          ...mockExporterBusinessTurnover,
```

Comment:
No need for the spread operator in this test file - duplicated below 

Could also move to a `validBody` const